### ISSUE TITLE
Transitioning.View doesnt work with Image on iOS 

### ISSUE THREAD

**Issue by mi5ha** [software-mansion/react-native-reanimated#681]
Only on iOS image is not animated properly.

I am using Animated.Image inside Transitioning.View

    "react": "16.9.0",
    "react-native": "0.61.5",
    "react-native-gesture-handler": "^1.6.1",
    "react-native-reanimated": "^1.7.0",

LayoutAnimation works properly btw.

Android works:
![android](https://user-images.githubusercontent.com/1238203/77545557-e1cc9c00-6eaa-11ea-8421-fef875f96959.gif)

iOS doesnt work:
![ios](https://user-images.githubusercontent.com/1238203/77545418-aaf68600-6eaa-11ea-9e7c-e7062d8d7879.gif)

``

**Comment by victorrseloy** [comment]
it seems that the underlying cause of problem is the same thing that is causing #657

**Comment by jakub-gonet** [comment]
We discussed Transitions internally in our team. Considering that v2 makes creating animations easier and Transitions was an experimental API for a long time, we decided to leave Transitions as they are.
 
We won’t actively fix bugs in them, but we’ll happily merge any PR with fixes. We’re also thinking about rewriting Transitions in v2 sometime in the future.

I’m closing this issue, but if you have further questions, I will be happy to help.
